app/renderer/components/messageBox.js

```diff
@@ -0,0 +1,165 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const React = require('react')
+const ImmutableComponent = require('../../../js/components/immutableComponent')
+const Dialog = require('../../../js/components/dialog')
+const Button = require('../../../js/components/button')
+const SwitchControl = require('../../../js/components/switchControl')
+const appActions = require('../../../js/actions/appActions')
+const KeyCodes = require('../../common/constants/keyCodes')
+const config = require('../../../js/constants/config')
+const {makeImmutable} = require('../../common/state/immutableUtil')
+const {StyleSheet, css} = require('aphrodite')
+const commonStyles = require('./styles/commonStyles')
+
+class MessageBox extends ImmutableComponent {
+  constructor () {
+    super()
+    this.onClick = this.onClick.bind(this)
+    this.onKeyDown = this.onKeyDown.bind(this)
+    this.onSuppressChanged = this.onSuppressChanged.bind(this)
+  }
+
+  componentWillMount () {
+    document.addEventListener('keydown', this.onKeyDown)
+  }
+  componentWillUnmount () {
+    document.removeEventListener('keydown', this.onKeyDown)
+  }
+
+  get tabId () {
+    return this.props.tabId || ''
+  }
+
+  get title () {
+    const msgBoxTitle = (this.props.detail && this.props.detail.get('title')) || ''
+    return msgBoxTitle.replace(config.braveExtensionId, 'Brave')
+  }
+
+  get message () {
+    return (this.props.detail && this.props.detail.get('message')) || ''
+  }
+
+  get buttons () {
+    return (this.props.detail && this.props.detail.get('buttons')) || makeImmutable(['OK'])
```

Comment:
Yeah I notice that button component has I10n-Id, but when I changed to the another language there was still the same string.